model/distro/db.go

```diff
@@ -123,7 +132,7 @@ func (d *Distro) Insert(ctx context.Context) error {
 
```

Comment:
Seems like `Insert` on L120 also needs to use the shared DB when it's available.

Reply:
thank you. Fixed